lighter/ws_client.py

```diff
@@ -51,6 +50,9 @@ def on_message(self, ws, message):
             self.handle_subscribed_account(message)
         elif message_type == "update/account_all":
             self.handle_update_account(message)
+        elif message_type == "ping":
```

Comment:
Not needed, as you handle the `ping` case below.

Reply:
I think we need this when using synchronous run without `asyncio`.